src/routes/+layout.svelte

```diff
@@ -385,12 +357,13 @@
                         {/each}
                     </div>
 
-                    <div class="relative mt-[2.8125vw] flex flex-col items-center gap-[0.75vw]">
-                        <active_glider
-                            bind:this={hover_glider_element}
-                            class="absolute h-[4vw] w-[4vw] rounded-[0.75vw] bg-white/10 opacity-0 ease-in-out duration-{GLIDER_TRANSITION_DURATION}"
-                        />
-
+                    <VercelHover
+                        direction="vertical"
+                        glider_container_class="mt-[2.8125vw] flex flex-col items-center gap-[0.75vw]"
```

Comment:
add `relative` class here?

Reply:
Thats what i dont wanna deal with. There will be a nest of problem down the line.